examples/darwin-framework-tool/commands/common/ControllerStorage.mm

```diff
@@ -143,6 +143,20 @@ - (void)storeValue:(NSData *)value forKey:key
     self.storage[controllerKey] = value;
 }
 
+- (void)print
+{
+    NSLog(@"%@ (%@)", kDarwinFrameworkToolControllerDomain, _keyScopingPrefix);
+    for (NSString * controllerKey in self.storage) {
+        if (![self _isControllerScopedKey:controllerKey]) {
+            continue;
+        }
+
+        __auto_type * key = [self _controllerScopedKeyToKey:controllerKey];
+        __auto_type * data = [self.storage objectForKeyedSubscript:controllerKey];
```

Comment:
Shouldn't that just use `objectForKey`?

Reply:
> Shouldn't that just use `objectForKey`?

No. But that is true that I can simplify it: `__auto_type * data = self.storage[controllerKey]`

Reply:
I have pushed an updated version.